src/lib/types/process.d.ts

```diff
@@ -0,0 +1,13 @@
+declare namespace Process {
```

Comment:
this will be odd to use like Process.Process can we rename to something else? maybe Payload?